src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingLumberjack.java

```diff
@@ -223,7 +223,26 @@ public void readFromNBT(@NotNull final NBTTagCompound compound)
                 treesToFell.put(new ItemStorage(stack), cut);
             }
         }
-        if (compound.hasKey(TAG_REPLANT))
+		else 
+		{
+			// We have a list of sapplings already, but we nee to load in the
+			// NBT file.
+			// check to see if it exist and if so then update it. Otherwise skip
+			// it,
+			// as it doesn't exist in minecraft anymore.
+			final NBTTagList saplingTagList = compound.getTagList(TAG_SAPLINGS, Constants.NBT.TAG_COMPOUND);
+			for (int i = 0; i < saplingTagList.tagCount(); ++i) {
+				final NBTTagCompound saplingCompound = saplingTagList.getCompoundTagAt(i);
+				final ItemStack stack = new ItemStack(saplingCompound);
```

Comment:
I think we can even only remove this whole else, and then remove the if(isEmpty) and add the part you did:

if (treesToFell.containsKey(storage)) 
{
	treesToFell.put(new ItemStorage(stack), cut);
}